src/addagent/validate.c

```diff
@@ -515,19 +519,83 @@ double OS_AgentAntiquity_ID(const char *id) {
     return ret;
 }
 
-/* Returns the number of seconds since last agent connection, or -1 if error. */
-double OS_AgentAntiquity(const char *name, const char *ip)
-{
-    struct stat file_stat;
-    char file_name[OS_FLSIZE];
+#ifdef CLIENT
+double OS_AgentAntiquity(const char *name, const char *ip){;}
 
-    snprintf(file_name, OS_FLSIZE - 1, "%s/%s-%s", AGENTINFO_DIR, name, ip);
+#else
 
-    if (stat(file_name, &file_stat) < 0)
-        return -1;
-
-    return difftime(time(NULL), file_stat.st_mtime);
+/**
+ * @brief Returns the number of seconds since last agent connection with a Query to Wazuh DB
```

Comment:
Not sure whether is necessary to mention how this method internally works. So probably the best description would be `Returns the number of seconds since the last agent connection.`

Reply:
Done!